nextcloudappstore/core/templates/base.html

```diff
@@ -9,6 +9,7 @@
     <meta name="author" content="">
     <link rel="shortcut icon" href="{% static 'assets/img/favicon.png' %}"/>
     <title>Nextcloud App Store{% block head-title %}{% endblock %}</title>
```

Comment:
Moved to https://github.com/nextcloud/appstore/issues/125 since it touches more places than this issue